uk.co.compucorp.civicrm.hrcore/js/src/hrcore/analytics/page-tracker.js

```diff
@@ -0,0 +1,37 @@
+/* global dataLayer */
+/* eslint-env amd */
+
+(function (window, CRM, dataLayer) {
+  define(function () {
+    'use strict';
+
+    return trackContactTabVirtualPageviews;
```

Comment:
it's kinda pointless, just return the function directly
```js
return function () {

}
```

Reply:
same for the helper icon function

Reply:
Not anymore. We now return objects.